swap/src/asb/event_loop.rs

```diff
@@ -350,11 +350,11 @@ where
         match self.db.insert_peer_id(swap_id, bob_peer_id).await {
             Ok(_) => {
                 if let Err(error) = self.swap_sender.send(swap).await {
-                    tracing::warn!(%swap_id, "Swap cannot be spawned: {}", error);
+                    tracing::warn!(%swap_id, "Failed to start swap: {}", error);
                 }
             }
             Err(error) => {
-                tracing::warn!(%swap_id, "Unable to save peer-id, swap cannot be spawned: {}", error);
+                tracing::warn!(%swap_id, "Unable to save peer-id in database: {}", error);
```

Comment:
```suggestion
                tracing::warn!(%swap_id, "Unable to save peer-id in database: {:#}", error);
```

Reply:
It is not an `anyhow::Error` so I don't think the alternate `Display` is going to do anything useful.